### PR TITLE
WebApi + React App 

### PR DESCRIPTION
**Pull request by bbarber**
A lot of boilerplate code to get up and running with a Database / Api / React site. The existing prototype site has been moved to a `prototype` directory and will be deleted once it is no longer useful

Screenshot to illustrate basic site layout and functionality
image

Repo structure
image

API structure
image

Site structure
image

### CODE REVIEW

**Comment by bbarber** [comment]
Sample solution contains a dummy end-to-end test with "Todo" items. This will be used to test our QA deployments, to ensure we have a valid test from UI -> API -> Database. This will get removed after testing

File: pipelines/on-merge-angular-develop.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+trigger:
```
Comment (Rhipeus) [inline pipelines/on-merge-angular-develop.yml]: I don't believe we need this angular file from the template.  Can we ditch it?

File: pipelines/on-merge-angular-staging.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+trigger:
```
Comment (Rhipeus) [inline pipelines/on-merge-angular-staging.yml]: I don't believe we need this angular file from the template.  Can we ditch it?

File: pipelines/pr-angular.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+trigger:
```
Comment (Rhipeus) [inline pipelines/pr-angular.yml]: I don't believe we need this angular file from the template.  Can we ditch it?

**Review by Rhipeus — CHANGES_REQUESTED** [review]
Also, namespace change to WesternStatesWater.WestDaat

**Comment by bbarber** [comment]
image

I am blessed to merge my own PR

File: pipelines/pr-angular.yml (hunk 3 above)
Comment (bbarber) [reply pipelines/pr-angular.yml:1]: Ditched in next PR

**Pull request merged**